changelog.md

```diff
@@ -1,4 +1,7 @@
 # Changelog - chrome.dart
+## 0.8.0-dev1 2015-09-15
+- updated the APIs to M44
```

Comment:
It is usually worth mentioning new apis, deprecated apis, new types we understand etc.